data/sql_partition/partition_schedule_a.sql

```diff
@@ -0,0 +1,259 @@
+    sa.amndt_ind_desc AS action_cd_desc,
+        CASE
+            WHEN "substring"(sa.sub_id::character varying::text, 1, 1) = '4'::text THEN sa.tran_id
+            ELSE NULL::character varying
+        END AS tran_id,
+        CASE
+            WHEN "substring"(sa.sub_id::character varying::text, 1, 1) = '4'::text THEN sa.back_ref_tran_id
+            ELSE NULL::character varying
+        END AS back_ref_tran_id,
+    sa.back_ref_sched_nm,
+    'SA' AS schedule_type,
+    sa.form_tp_desc AS schedule_type_desc,
+    sa.line_num,
+    sa.image_num,
+    sa.file_num,
+    sa.link_id,
+    sa.orig_sub_id,
+    sa.sub_id,
+    vs.form_tp_cd AS filing_form,
+    vs.rpt_tp,
+    vs.rpt_yr,
+    vs.rpt_yr + vs.rpt_yr % 2::numeric AS election_cycle
+   FROM disclosure.nml_sched_a sa,
+    disclosure.v_sum_and_det_sum_report vs
+  WHERE sa.link_id = vs.orig_sub_id AND sa.amndt_ind::text <> 'D'::text AND sa.delete_ind IS NULL;
+
+---- Insert the records from the view
+INSERT INTO ofec_sched_a_master_tmp SELECT
+    cmte_id,
+    cmte_nm,
+    contbr_id,
+    contbr_nm,
+    contbr_nm_first,
+    contbr_m_nm,
+    contbr_nm_last,
+    contbr_prefix,
+    contbr_suffix,
+    contbr_st1,
+    contbr_st2,
+    contbr_city,
+    contbr_st,
+    contbr_zip,
+    entity_tp,
+    entity_tp_desc,
+    contbr_employer,
+    contbr_occupation,
+    election_tp,
+    fec_election_tp_desc,
+    fec_election_yr,
+    election_tp_desc,
+    contb_aggregate_ytd,
+    contb_receipt_dt,
+    contb_receipt_amt,
+    receipt_tp,
+    receipt_tp_desc,
+    receipt_desc,
+    memo_cd,
+    memo_cd_desc,
+    memo_text,
+    cand_id,
+    cand_nm,
+    cand_nm_first,
+    cand_m_nm,
+    cand_nm_last,
+    cand_prefix,
+    cand_suffix,
+    cand_office,
+    cand_office_desc,
+    cand_office_st,
+    cand_office_st_desc,
+    cand_office_district,
+    conduit_cmte_id,
+    conduit_cmte_nm,
+    conduit_cmte_st1,
+    conduit_cmte_st2,
+    conduit_cmte_city,
+    conduit_cmte_st,
+    conduit_cmte_zip,
+    donor_cmte_nm,
+    national_cmte_nonfed_acct,
+    increased_limit,
+    action_cd,
+    action_cd_desc,
+    tran_id,
+    back_ref_tran_id,
+    back_ref_sched_nm,
+    schedule_type,
+    schedule_type_desc,
+    line_num,
+    image_num,
+    file_num,
+    link_id,
+    orig_sub_id,
+    sub_id,
+    filing_form,
+    rpt_tp,
+    rpt_yr,
+    election_cycle,
+    CURRENT_TIMESTAMP as timestamp,
+    CURRENT_TIMESTAMP as pg_date,
+    image_pdf_url(image_num) AS pdf_url,
+    to_tsvector(concat(contbr_nm, ' ', contbr_id)) AS contributor_name_text,
+    to_tsvector(contbr_employer) AS contributor_employer_text,
+    to_tsvector(contbr_occupation) AS contributor_occupation_text,
+    is_individual(contb_receipt_amt, receipt_tp, line_num, memo_cd, memo_text) AS is_individual,
+    clean_repeated(contbr_id, cmte_id) AS clean_contbr_id,
+    get_transaction_year(contb_receipt_dt, rpt_yr) AS two_year_transaction_period
+FROM fec_vsum_sched_a_vw;
```

Comment:
We want to insert from the existing `fec_fitem_sched_a_vw` view, which is created and managed by the FEC.  I'm sorry for the confusion!